api/src/templates/gift_card_purchase.subject.html

```diff
@@ -0,0 +1 @@
+Gift card purchased
```

Comment:
This subject sounds a bit like a receipt. But we already send that in a separate email, right?

So I guess this email is supposed to send some more details about gift card that was purchased. Like how to get access to it, or what validation code it has. In that case I think this email subject should rather be something like "Here is your gift card" or similar.